database/layer/knex.js

```diff
@@ -1,5 +1,16 @@
 const knex = require('knex');
 const knexDbConfig = require('../knexfile');
+
+/**
+ * Usher DB connection instance.
```

Comment:
@mikelax 

![image](https://github.com/DMGT-TECH/the-usher-server/assets/19498202/3a92ba93-27e5-4163-8c70-93abf37928a8)